packages/tupaia-web/src/features/Map/utils/useMapOverlayData.ts

```diff
@@ -49,7 +51,7 @@ const getRootEntity = (entity?: Entity) => {
   return parentCode;
 };
 
-export const useActiveMapOverlayReport = () => {
+export const useMapOverlayData = (hiddenValues?: LegendProps['hiddenValues']) => {
```

Comment:
I see that `hiddenValues` is a param for this. But am I right in saying nowhere passes in `hiddenValues`? If so, should somewhere be doing that?

Reply:
@WDuncker just bumping this one 🙏